perun-base/src/main/java/cz/metacentrum/perun/rpc/serializer/JsonSerializerJSONLITE.java

```diff
@@ -0,0 +1,234 @@
+		@JsonSerialize
+		@JsonProperty(value = "startTime")
+		Long getStartTimeAsLong();
+
+		@JsonIgnore
+		LocalDateTime getStartTime();
+
+		@JsonSerialize
+		@JsonProperty(value = "schedule")
+		Long getScheduleAsLong();
+
+		@JsonIgnore
+		LocalDateTime getSchedule();
+
+		@JsonSerialize
+		@JsonProperty(value = "genEndTime")
+		Long getGenEndTimeAsLong();
+
+		@JsonIgnore
+		LocalDateTime getGenEndTime();
+
+		@JsonSerialize
+		@JsonProperty(value = "sendEndTime")
+		Long getSendEndTimeAsLong();
+
+		@JsonIgnore
+		LocalDateTime getSendEndTime();
+
+		@JsonSerialize
+		@JsonProperty(value = "sendStartTime")
+		Long getSendStartTimeAsLong();
+
+		@JsonIgnore
+		LocalDateTime getSendStartTime();
+
+		@JsonSerialize
+		@JsonProperty(value = "genStartTime")
+		Long getGenStartTimeAsLong();
+
+		@JsonIgnore
+		LocalDateTime getGenStartTime();
+
+		@JsonSerialize
+		@JsonProperty(value = "sentToEngine")
+		Long getSentToEngineAsLong();
+
+		@JsonIgnore
+		LocalDateTime getSentToEngine();
+
+		@JsonSerialize
+		@JsonProperty(value = "endTime")
+		Long getEndTimeAsLong();
+
+		@JsonIgnore
+		LocalDateTime getEndTime();
+	}
+
+	public static final String CONTENT_TYPE = "application/json; charset=utf-8";
+	private static final ObjectMapper mapper = new ObjectMapper();
+	private static final Map<Class<?>,Class<?>> mixinMap = new HashMap<>();
+
+	static {
+		mixinMap.put(Attribute.class, AttributeMixIn.class);
+		mixinMap.put(AttributeDefinition.class, AttributeDefinitionMixIn.class);
+		mixinMap.put(User.class, UserMixIn.class);
+		mixinMap.put(Candidate.class, CandidateMixIn.class);
+		mixinMap.put(PerunException.class, ExceptionMixIn.class);
+		mixinMap.put(PerunRuntimeException.class, ExceptionMixIn.class);
+		mixinMap.put(PerunBean.class, PerunBeanMixIn.class);
+		mixinMap.put(PerunRequest.class, PerunRequestMixIn.class);
+		mixinMap.put(Authorship.class, CabinetMixIn.class);
+		mixinMap.put(Author.class, CabinetMixIn.class);
+		mixinMap.put(Category.class, CabinetMixIn.class);
+		mixinMap.put(Publication.class, CabinetMixIn.class);
+		mixinMap.put(Thanks.class, CabinetMixIn.class);
+		mixinMap.put(Task.class, TaskMixIn.class);
+
+		mapper.setMixIns(mixinMap);
+	}
+
+	private static final JsonFactory jsonFactory = new JsonFactory();
+
+	static {
+		//FIXME odstraneno disable(JsonGenerator.Feature.FLUSH_PASSED_TO_STREAM)
+		jsonFactory
+			.disable(JsonGenerator.Feature.AUTO_CLOSE_TARGET)
+			.disable(JsonGenerator.Feature.AUTO_CLOSE_JSON_CONTENT)
+			.setCodec(mapper);
+	}
+
+	private OutputStream out;
+
+	/**
+	 * @param out {@code OutputStream} to output serialized data
+	 * @throws IOException if an IO error occurs
+	 */
+	public JsonSerializerJSONLITE(OutputStream out) throws IOException {
+		this.out = out;
+	}
+
+	@Override
+	public String getContentType() {
+		return CONTENT_TYPE;
+	}
+
+	@Override
+	public void write(Object object) throws IOException {
+		JsonGenerator gen = jsonFactory.createJsonGenerator(out, JsonEncoding.UTF8);
```

Comment:
Method "jsonFactory.createJsonGenerator()" is deprecated. Is there any chance to change that without great effort?

Reply:
Done.